src/components/SelectComponents/Autosuggest/Autosuggest.tsx

```diff
@@ -43,6 +43,8 @@ interface Props<S>
     /** Means to overwrite the look and feel of the UI in its blurred state.
      * It should be null or undefined if there is no selection yet */
     customSelectionIndicator?: ReactNode;
+    /** Additional HTML attributes to be applied to the input element */
+    inputAttr?: DictionaryOf<string | boolean>;
```

Comment:
Let's use `React.InputHTMLAttributes<HTMLInputElement>`

Reply:
I tried but that won't allow for arbitrary `data-something` attributes which is what we need here

Reply:
I thought `React.InputHTMLAttributes<HTMLInputElement>` supports data attribute :( 
I found some solution, but it seems only proposal https://github.com/microsoft/TypeScript/pull/21044.
Why do we need to support boolean?

Reply:
boolean for data attributes... I don't think there is a simple way to pass them without adding the value